packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -164,6 +164,7 @@ export function SelectWalletModal({
       };
       connectToUnisatWalletOnLoad();
     }
+    return () => window.unisat.removeListener("accountsChanged", listener);
```

Comment:
this is adding/removing listener side effects in 2 different areas which ideally should be co-located together in the same effect. let's constrain it to one useEffect so that it's clear that the same instance is being added and removed.

I think we can remove the window.unisat.add/remove listener calls from `onConnectUnisatWallet` and let this effect manage it.